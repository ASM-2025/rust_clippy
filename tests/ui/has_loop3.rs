#![allow(unused)]
#![warn(clippy::has_loop)]

fn main() {
    let mut i = 4;
    while i < 10 {
        println!("{}", i);
        i += 1;
    }
}
