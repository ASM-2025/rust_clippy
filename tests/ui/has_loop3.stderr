error: Loop detected
  --> $DIR/has_loop3.rs:6:5
   |
LL | /     while i < 10 {
LL | |         println!("{}", i);
LL | |         i += 1;
LL | |     }
   | |_____^
   |
   = note: `-D clippy::has-loop` implied by `-D warnings`

error: aborting due to previous error

